dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/HttpClientUtil.java

```diff
@@ -43,19 +45,11 @@ protected JSSEHelper getJSSEHelper(SSLSupport sslSupport) throws SSLException {
 
```

Comment:
missed an ffdc ignore 😅

```suggestion
@FFDCIgnore(NoSSLSocketFactoryException.class)
```